runtime-data/StakingToken.json

```diff
@@ -0,0 +1,750 @@
+{
```

Comment:
I understand why this is not under `accounts` but maybe you could place `accounts` in `runtime-data` now since there are at least two categories of JSON files that test scripts use (at runtime): keys and the staking token. There's also `parity-data` which is populated by files generated when the test runs. I think we need a separate directory for static data, like keys and token JSONs. I'm not sure about the name. Possibly not `runtime-data` because everything in it is going to be static. Maybe `static-data` or `test-data`?

Reply:
Actually, I didn't want it to be here and forgot to add it into gitignore!
My idea was that these files will be generated by `deploy-staking-token.js` when network starts, you can actually see in `deploy-staking-token.sh` that this folder is cleared when network starts.

The caveat that I'm trying to avoid by doing it this way, is that contract's address may (in principle) change, if OWNER makes any other transactions before deploying it, because contract address is calculated from deployer's account and account nonce.

Does it make sense?

Reply:
So `runtime-data` folder is intended for any information shared between tests, but specific to a particular test run, something that cannot be moved to `utils/constants.js`

Reply:
Would you delete the contents of `runtime-data` after each test?

Reply:
Ok, moving it into parity-data alongside spec.json makes more sense :+1: